docs/_static/spec/action_server.yml

```diff
@@ -0,0 +1,365 @@
+openapi: "3.0.2"
+info:
+  title: "Rasa Core SDK - Action Server Endpoint"
+  version: "1.0.0"
+  description: >-
+    API of the action server which is used by Rasa Core 
+    to execute custom actions.
+paths:
+  /webhook:
+    post:
+      summary: Call a custom action
+      description: >-
+        Sends a request to the action server to execute a custom action.
+      operationId: call_action
+      requestBody:
+        description: >- 
+          Describes the action to be called and provides information on the 
+          current state of the conversation.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                next_action:
+                  description: The name of the action which should be executed.
+                  type: string
+                sender_id:
+                  description: >-
+                    Id of the user who is having the current conversation.
+                  type: string
+                tracker:
+                  $ref: "#/components/schemas/Tracker"
+                domain:
+                  $ref: "#/components/schemas/Domain"
+      responses:
+        200:
+          description: Action was executed succesfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  events:
+                    description: "Events returned by the action."
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Event"
+                  responses:
+                    description: >-
+                      "List of responses which should be sent to the user"
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Response"
+        400:
+          description: >-
+            Action execution was rejected. This is the same as returning
+            an `action_execution_rejected` event.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  action_name:
+                    type: string
+                  error:
+                    type: string
+        500:
+          description: >-
+            The action server encountered an exception while running the action.
+components:
```

Comment:
I think at least some of these are duplicated between the two specifications - how about creating a common file we can import from: seems to be possible to ref components in different files: https://swagger.io/docs/specification/using-ref/
and 
https://dev.to/mikeralphson/defining-reusable-components-with-the-openapi-specification-4077